src/Symfony/Component/Translation/Bridge/Loco/Tests/LocoProviderFactoryWithoutTranslatorBagTest.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Symfony\Component\Translation\Bridge\Loco\Tests;
+
+use Symfony\Component\Translation\Bridge\Loco\LocoProviderFactory;
+use Symfony\Component\Translation\Provider\ProviderFactoryInterface;
+
+/**
+ * This test should be removed in Symfony 7.0 when making $translatorBag argument not nullable for the "LocoProviderFactory" constructor.
+ *
+ * @group legacy
+ */
+class LocoProviderFactoryWithoutTranslatorBagTest extends LocoProviderFactoryTest
+{
+    public function createFactory(): ProviderFactoryInterface
+    {
+        return new LocoProviderFactory($this->getClient(), $this->getLogger(), $this->getDefaultLocale(), $this->getLoader());
```

Comment:
Shall we test if the deprecation is actually triggered?

Reply:
How can I achieve that?